Cart/cart.css

```diff
@@ -0,0 +1,27 @@
+.div {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 5px;
+}
+
+.cart-btn {
+  width: 100%;
+}
+
+.cart-text {
+  padding: 5px 0;
+}
+
```

Comment:
Use of rem is preferred

Reply:
Vikram mentioned below 10px we need not convert to rem